libr/util/unum.c

```diff
@@ -672,18 +677,18 @@ R_API ut64 r_get_input_num_value(RNum *num, const char *input_value){
 	return value;
 }
 
-static bool isHexDigit (const char _ch) {
-	const char ch = tolower ((ut8)_ch);
-	if (IS_DIGIT (ch)) {
-		return true;
-	}
-	return (ch >= 'a' && ch <= 'f');
-}
-
+/**
+ * \brief Get i_th nibble from integer n
```

Comment:
rename the function if its not clear what it does instead of adding 8 lines of comments to explain 3 lines of C.